gradle-android-command-plugin/src/main/groovy/com/novoda/gradle/command/Monkey.groovy

```diff
@@ -15,12 +16,29 @@ class Monkey extends AdbTask {
 
     @TaskAction
     void exec() {
-        def arguments = ['shell', 'monkey', '-p', packageName, '-v', getEvents()]
+        def arguments = ['shell', 'monkey', '-p', packageName]
```

Comment:
I might just split the '-p' and packageName ones out onto the next line too, for consistency